contents/blog/posthog-cloud-eu.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2022-10-10
+title: Introducing PostHog Cloud EU
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Product updates”]
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-eu-blog.png
```

Comment:
We need to change this image. The blue doesn't seem the right share, there's transparent shadow left on the image and the fill doesn't alias correctly. @lottiecoxon is off ill today and, since we need this to be ready for tomorrow, I'm going to suggest we swap this for an existing image. It'll be less targeted, but look better.

Reply:
```suggestion
featuredImage: ../images/blog/posthog-release-notes-blog.png
```

Reply:
Didn't realise Lottie was off. I cant probably do a better version of this, or maybe Cory can help? The blue is the exact shade of the EU flag, btw.

Reply:
Have updated the image so it looks ok. Something more custom would be preferable if Lottie is back tomorrow.